source/concepts/block.rst

```diff
@@ -59,6 +48,35 @@ Serialization of the block header.
 
    <a href="https://en.wikipedia.org/wiki/Radix_tree" target="_blank">patricia tree</a>
 
+********
+Creation
+********
+
+|codename| calls the first block in the chain the **nemesis block**.
+The first block is defined before launching a new network and sets the distribution of the network currency mosaics.
+
+The process of creating the subsequent new blocks is called :doc:`harvesting <harvesting>`.
+
+Blocks are created by :doc:`accounts <account>` based on their importance score.
+The :doc:`consensus algorithm <consensus-algorithm>` chooses each round the account that harvest the block. 
+
+The harvesting account receives the :doc:`fees <fees>` for the transactions added in the block and the mosaics created by :doc:`inflation <inflation>`.
+This gives the harvester an incentive to add as many transactions to the block as possible.
+
+.. _rollbacks:
+
+*********
+Rollbacks
+*********
+
+Blockchains are designed in a way that, in the presence of a network failure or partition, the recent blocks might need to be rolled back.
+
+The rewrite limit is the maximum number of blocks that can be rolled back.
+Hence, forks can only be resolved up to a certain depth too.
+
+|codename|'s public network has a rewrite limit of ``398`` blocks, being this limit :ref:`configurable per network <config-network-properties>`.
+At this point, the transactions linked to the block are permanently recorded on the blockchain.
```

Comment:
The transactions linked to a block are permanently recorded on the blockchain once the number of confirmation blocks (subsequent blocks) surpasses the maximum number of rollback blocks.